pkg/randomid/id_test.go

```diff
@@ -0,0 +1,12 @@
+package randomid
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenerate(t *testing.T) {
+	id := Generate()
+	assert.Equal(t, len(id), 64)
```

Comment:
Could you change this line of code to be `assert.Equal(t, len(id), 64)`. Because in pkg `github.com/stretchr/testify/assert`, we have:
```
func Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{}) bool {
```

Expected is the second parameter, and actual is the third.

Although, we can leave them there, while if CI fails, the output will make people confused, like the previous CI failure:
```
--- FAIL: TestGenerate (0.00s)
	Error Trace:	id_test.go:11
	Error:		Not equal: 64 (expected)
			        != 32 (actual)
		
FAIL
```
It mixed the actual and expected.

Reply:
Hi, the `assert.Equal(t, len(id), 64)` is exactly what I did. 

> Expected is the second parameter, and actual is the third.

So I should change it to `assert.Equal(t, 64, len(id))` right?